src/yarp.c

```diff
@@ -9773,6 +9773,12 @@ parse_arguments_list(yp_parser_t *parser, yp_arguments_t *arguments, bool accept
         // argument to this method call.
         parse_arguments(parser, arguments, true, YP_TOKEN_EOF);
 
+        // If we have done with the arguments and still not consumed the comma, then we have a trailing comma
+        // where we need to check whether it is allowed or not.
+        if (parser->previous.type == YP_TOKEN_COMMA && !match1(parser, YP_TOKEN_SEMICOLON)) {
```

Comment:
I think the check for semicolon here would allow `foo 1,;`. Is that intentional?